pkg/version/version_test.go

```diff
@@ -0,0 +1,28 @@
+//
+// Copyright 2022 The Sigstore Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package version
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/require"
+)
+
+func TestVersionText(t *testing.T) {
+	sut := GetVersionInfo()
+	require.NotEmpty(t, sut)
+	require.Equal(t, gitVersion, sut.GitVersion)
```

Comment:
```suggestion
	if sut.GitVersion != gitVersion {
		t.Errorf("GetVersionInfo: got %q, want %q", sut, gitVersion)
	}
```

We don't need to introduce testify just for this. WDYT?